README.md

```diff
@@ -44,6 +44,31 @@ To run the following examples you will also need these ROS 1 packages:
 * `rostopic`
 * `rqt_image_view`
 
+### Prerequisites for the examples in this file
+
+In order to make the examples below portable between versions of ROS, we've chosen to define two environment variables, `ROS1_INSTALL_PATH` and `ROS2_INSTALL_PATH`. 
+These are defined as the paths to the installation location of their respective ROS versions.
+
+If you installed Noetic in the default location, then the definition of `ROS1_INSTALL_PATH` would likely be `/opt/ros/noetic`.
+
+Building the bridge as described below requires you to build all of ROS 2.
+We assume that you have downloaded it to `~/ros2_galactic`, and that is where you plan on building it.
+In this case, `ROS2_INSTALL_PATH` will be defined as `~/ros2_galactic/install`.
+
+If you've chosen to install either or both versions of ROS somewhere else, you will need adjust the definitions below to match your installation paths.
+
+Because these definitions are used continuously throughout this file, it would be useful to copy the following lines to your `.bashrc` file (if you're using `bash` as your shell).
+Modify these definitions as appropriate for the versions of ROS that you're using, and for the shell that you're using.
```

Comment:
```suggestion
```

Reply:
I don't want to remove this line for two reasons:

- While `bash` and `zsh` are common because they are the defaults on Linux and OS X respectively, they aren't the only available shells.  I want to remind users that they need to think about their shells.
- When I first started working on this update, it talked about `melodic` and `foxy`.  That suggests that this documentation may live long enough that the latest and greatest version of ROS 2 will change.  I want to make sure that we're letting end users know that they need to think about what is going on behind the scenes, rather than blindly following the instructions.

Reply:
We have specific versions of the README for each ROS version. Which reminds me that this particular version, being `master`, should actually refer to ROS 2 Rolling and Noetic. The correct path is `~/ros2_rolling` and there is no need to instruct the user to alter the instructions for different ROS 2 versions.

Reply:
Understood, I'll modify it.